### PR TITLE
Add deprecation notice

### PR DESCRIPTION
**Pull request by torbsto**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,13 +4,17 @@
 ![Code Quality](https://github.com/bakdata/quick-cli/workflows/Code%20Quality/badge.svg)
 [![PyPI Version](https://img.shields.io/pypi/v/quick-cli?color=blue&label=PyPi%20Version)](https://pypi.org/project/quick-cli/)
 
+> [!WARNING]
+> Quick's CLI is deprecated and no longer actively maintained.
+> Visit our [user guide](https://bakdata.github.io/quick/0.8/user/) for more information.
```
Comment (chrboehm) [inline README.md:9]: better version 'latest' then 0.8 or dev? see also other links.

**Comment by chrboehm** [comment]
lgtm.

File: README.md (hunk 1 above)
Comment (raminqaf) [reply README.md:9]: Since we are not releasing any new versions, the `0.8` would be `latest`

**Review by raminqaf — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (torbsto) [reply README.md:9]: This link should remain at 0.8 because we might remove the warning someday

**Pull request merged**